### ISSUE TITLE
Release 0.1.6 fails to export to linux

### ISSUE THREAD

**Issue by youngm** [cloudfoundry/smb-volume-release#6]
If I try to export the smb-volume-release 0.1.6 to a linux stemcell I get the error below.  I'm not sure how other releases that have windows packages mixed in solve this.

Perhaps the windows package's packaging script can skip if exporting to a linux stemcell since the linxu stemcell won't be using the windows job anyway?

```
Task 107619 | 21:20:20 | Compiling packages: golang-smbvolume-windows/a979fc2219d6964af5f0ae38f1e9cd1df75e5325 (00:01:07)
                      L Error: Action Failed get_task: Task 22c10af0-a5e0-44bd-4473-633707b871d8 result: Compiling package golang-smbvolume-windows: Running packaging script: Running packaging script: Command exited with 2; Stdout: , Stderr: packaging: line 1: =: command not found
packaging: line 2: syntax error near unexpected token `('
packaging: line 2: `trap { $host.SetShouldExit(1) }'
```

**Comment by paulcwarren** [comment]
Hey @youngm it looks like you are trying to deploy windows job on linux stemcell.  How are you deploying this Mike?  Are you using `cf-deployment` or creating your own manifests?

**Comment by julian-hj** [comment]
This is resolved in v1.10.  Thanks @youngm !
